pom.xml

```diff
@@ -0,0 +1,244 @@
+				<plugin>
+					<groupId>org.eclipse.tycho.extras</groupId>
+					<artifactId>tycho-source-feature-plugin</artifactId>
+					<version>${tycho.extras.version}</version>
+				</plugin>
+				<plugin>
+					<groupId>org.eclipse.tycho</groupId>
+					<artifactId>tycho-source-plugin</artifactId>
+					<version>${tycho.version}</version>
+				</plugin>
+				<plugin>
+					<groupId>org.eclipse.tycho</groupId>
+					<artifactId>tycho-p2-plugin</artifactId>
+					<version>${tycho.version}</version>
+				</plugin>
+				<plugin>
+					<groupId>org.eclipse.tycho.extras</groupId>
+					<artifactId>tycho-dependency-tools-plugin</artifactId>
+					<version>1.7.0</version>
+					<executions>
+						<execution>
+							<id>list-dependencies</id>
+							<goals>
+								<goal>list-dependencies</goal>
+							</goals>
+							<phase>compile</phase>
+						</execution>
+					</executions>
+				</plugin>
+			</plugins>
+		</pluginManagement>
+
+		<plugins>
+			<plugin>
+				<!-- This plugin configuration block is only needed if the repository contains plug-ins that don't have any sources. If it is omitted
+		 		Tycho will complain. -->
+				<groupId>org.eclipse.tycho.extras</groupId>
+				<artifactId>tycho-source-feature-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>source-feature</id>
+						<phase>package</phase>
+						<goals>
+							<goal>source-feature</goal>
+						</goals>
+					</execution>
+				</executions>
+				<configuration>
+					<excludes>
+                        <plugin id="org.rdkit.knime.bin.linux.x86_64"/>
+                        <plugin id="org.rdkit.knime.bin.win32.x86_64"/>
+                        <plugin id="org.rdkit.knime.bin.macosx.x86_64"/>
+					</excludes>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-maven-plugin</artifactId>
+				<extensions>true</extensions>
+			</plugin>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-packaging-plugin</artifactId>
+				<dependencies>
+					<dependency>
+						<groupId>org.eclipse.tycho.extras</groupId>
+						<artifactId>tycho-buildtimestamp-jgit</artifactId>
+						<version>${tycho.extras.version}</version>
+					</dependency>
+				</dependencies>
+				<configuration>
+					<timestampProvider>jgit</timestampProvider>
+					<format>'v'yyyyMMddHHmm</format>
+					<jgit.ignore>
+						pom.xml
+						.polyglot.build.properties
+					</jgit.ignore>
+					<jgit.dirtyWorkingTree>warning</jgit.dirtyWorkingTree>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-p2-repository-plugin</artifactId>
+				<configuration>
+					<skipArchive>true</skipArchive>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>target-platform-configuration</artifactId>
+				<configuration>
+					<environments>
+						<!-- These can be removed once we shut down the Buckminster build -->
```

Comment:
Just curious: Why does Buckminster still play a role inside of a pom.xml file for Tycho? Don't we always have the situation to build environment specific packages with the RDKit binaries?

Reply:
That comment is actually wrong, this is required for multi-platform builds, I have to clean that up that from the template I copied this from
```suggestion
```